simplify new chapter (#859)
SUSE/doc-susemanager: modules/administration/pages/ssl-certs.adoc

== Replace Existing CA and Server Certificates with third party certificiate

> You can replace working certificates on an existing {productname} installation with a new third party certificate.

Reviewer: Drive-by edit:
```suggestion
You can replace active certificates on your {productname} installation with a new third party certificate.
```

> In this case, you can replace the old CA certificate RPM with the new one, and update the database.

Reviewer: Drive-by edit:
```suggestion
To replace the certificates, you can replace the installed CA certificate RPM with a new RPM containing the third party certificate, and then update the database.
```

> This procedure is similar to the one described in "Create and Replace CA and Server Certificates" with the difference, that we import the certificates generated by an external PKI.

Reviewer: proper link wanted. we must create a section label first.
